### PR TITLE
init route-auth

### PR DESCRIPTION
**Pull request by StoicJester**
@jrit split route-auth off from ng-lib. No changes to the code itself. Most significant change is to the README.md file

### CODE REVIEW

File: .jshintignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+next-tier/static/scripts/tests/*
```
Comment (jrit) [inline .jshintignore]: shouldn't be here

File: README.md [hunk 2]
```diff
@@ -1 +1,157 @@
...
+* [Usage](#section-usage)
+* [Interface](#section-interface)
+
+## <a name="section-usage"></a>Usage
+
+Include the file found in the `dist` directory.
+
+When creating your main angular module, include `vokal.majora` in the list of included modules, e.g.
+```js
+angular.module( "myApp", [ "vokal.majora" ] );
```
Comment (jrit) [inline README.md]: shouldn't say majora here

File: README.md [hunk 3]
```diff
@@ -1 +1,157 @@
...
+Overwrite the user's current roles with `newRoles`
+
+##### Arguments
+
+1. `newRoles` | *Array* | the new user roles to be set
+
+##### Example
+
+```js
+RouteAuth.storeRoles( [ 'user', 'premiumUser', 'purpleDiamondUltraEliteClass' ] );
```
Comment (jrit) [inline README.md:66]: Have you flown United recently?

File: README.md [hunk 4]
```diff
@@ -1 +1,157 @@
...
+* * *
+
+#### <a name="method-auth"></a>`auth( allowedRoles, options )`
+
+Returns a promise, which is resolved if the user has one of the `allowedRoles`. Otherwise, the promise is rejected.
+ 
+##### Arguments
+
+1. `allowedRoles` | *Array*  | list of acceptable roles
+2. `options`      | *Object* | optional options for this function
```
Comment (jrit) [inline README.md]: This should say what the options are, also `optional options` is really funny

Comment (jrit) [inline README.md]: oh, it does say below, sorry, I'm reading the markdown

File: bower.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "route-auth",
+  "homepage": "https://github.com/vokal/route-auth",
+  "authors": [
+    {
+      "name": "VOKAL Interactive",
```
Comment (jrit) [inline bower.json]: "Vokal"

**Comment by jrit** [comment]
A few typos, then LGTM. Thanks for taking care of this.

File: README.md (hunk 3 above)
Comment (StoicJester) [inline README.md:66]: No, but read a joke about flight classes and felt inspired.

**Comment by StoicJester** [comment]
fixes made

**Comment by jrit** [comment]
Can you just add a version 1.0.0 to package.json, then I'll merge?

**Comment by StoicJester** [comment]
like that?

**Comment by jrit** [comment]
yup

**Comment by Tathanen** [comment]
Didn't we decide we didn't need versions in package.json?

**Comment by jrit** [comment]
No, that is bower.json. npm uses the version info but bower doesn't because it uses git tags.

**Comment by Tathanen** [comment]
Are we... ever going to npm install this?

**Comment by jrit** [comment]
We could, only requires creating on npm and then we could install by name. We'll need to coordinate on some account info for npm though because of how it works. I'll add that to my to dos.

**Pull request merged**